src/test/java/org/fundacionjala/sfdc/stepdefinition/AccountSteps.java

```diff
@@ -0,0 +1,118 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.commons.PropertiesManager;
+import org.fundacionjala.sfdc.pageobjects.Accounts.SFADetailsPage;
+import org.fundacionjala.sfdc.pageobjects.Accounts.SFAMainPage;
+import org.fundacionjala.sfdc.pageobjects.Accounts.SFANewModifyPage;
+import org.fundacionjala.sfdc.pageobjects.SalesForceMainTabClassic;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import java.util.Properties;
+
+/**
+ * AccountSteps.java
+ * Class for steps of account.feature.
+ */
+public class AccountSteps {
+    private WebDriver webDriver;
+    private Properties propertiesManager;
+
+    /**
+     * Given step.
+     */
+    @Given("^Go to accounts section$")
+    public void goToAccountsSection() {
+        webDriver = DriverManager.getInstance().getNavigator();
+        SalesForceMainTabClassic salesForceMainTab;
+        salesForceMainTab = new SalesForceMainTabClassic(webDriver, "a.accountBlock", "li#AllTab_Tab");
+        salesForceMainTab.displayOptions();
+        salesForceMainTab.goToAccounts();
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I press new account button a new account form is displayed$")
+    public void iPressNewAccountButtonANewAccountFormIsDisplayed() {
+        SFAMainPage accountsMainPage = new SFAMainPage();
+        accountsMainPage.clickNewAccountButton();
+    }
+
+    /**
+     * And step.
+     */
+    @And("^I fill the account name field and press the save button$")
+    public void iFillTheAccountNameFieldAndPressTheSaveButton() {
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        SFANewModifyPage newAccountPage = new SFANewModifyPage();
+        newAccountPage.setAccountNameTextField(propertiesManager.getProperty("accountName"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "accountName" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)